src/Modal/Modal.style.js

```diff
@@ -0,0 +1,156 @@
+import React from "react";
+import styled, { createGlobalStyle, css } from "styled-components";
+import { fontFamily, boxSizing, color, typography } from "@bonio-ui/theme";
+import { Cross1 } from "@bonio-ui/icons";
+import media from "../utils/media";
+
+const { mask, white, gray, hover } = color;
+const { title3, body2 } = typography;
+
+export const noScroll = "no-scroll";
+
+export const GlobalStyle = createGlobalStyle`
+  .${noScroll} {
+    overflow: hidden;
+  }
+`;
+
+export const Container = styled.div`
+  ${({ open }) => !open && "display: none;"}
+
+  ${fontFamily}
+  ${boxSizing}
+
+  position: fixed;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  background: ${mask[2]};
+  z-index: 2147483646;
+  overflow-y: auto;
+
+  *,
+  *::before,
+  *::after {
+    box-sizing: border-box;
+  }
+`;
+
+const size = {};
+size.medium = css`
+  ${media.tablet} {
+    padding-left: 80px;
+    padding-right: 80px;
+  }
+`;
+size.large = css`
+  ${media.tablet} {
+    padding-left: 56px;
+    padding-right: 56px;
+  }
+
+  ${media.desktop} {
+    max-width: 900px;
+    padding-left: 80px;
+    padding-right: 80px;
+  }
+`;
+
+export const Content = styled.div`
+  position: fixed;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  width: 100%;
+  max-width: 640px;
+  max-height: 100%;
+  padding-top: 56px;
+  padding-bottom: 40px;
+  padding-left: 35px;
+  padding-right: 35px;
+  border-radius: 10px;
+  background: ${white};
+  box-shadow: 0 2px 10px rgba(0, 0, 0, 0.12);
+  overflow-y: auto;
+  -webkit-overflow-scrolling: touch;
+
+  ${(props) => size[props.size]}
+`;
+
+export const TitleWrapper = styled.div`
+  padding-bottom: 16px;
+  text-align: center;
+`;
+
+export const Title = styled.header`
+  ${title3}
+`;
+
+export const Subtitle = styled.div`
+  margin-top: 8px;
+  ${body2}
+`;
+
+export const ActionWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  padding-top: 24px;
+
+  button {
+    width: 250px;
+    max-width: 100%;
+  }
+
+  button + button {
+    margin-top: 8px;
+    margin-left: 0;
+  }
+
+  ${media.tablet} {
+    flex-direction: row;
+    justify-content: flex-end;
+
+    button {
+      width: auto;
+      min-width: 120px;
+    }
+
```

Comment:
接續上個問題,
避免桌面版本受到行動版的影響。
```suggestion
    button:nth-child(1) {
      margin-top: unset;
      margin-left: unset;
  }
```